### DISCUSSION TITLE
Add name property to q-btn and post it in q-form

### DISCUSSION THREAD

**Discussion by JTrotta** [quasarframework/quasar#16163] [Ideas / Proposals] [answered]
Using standard <form> tag and a q-btn as submit button, when the form is post among all the form keys there is also the q-btn name. 
Instead using <q-form> tag and a q-btn as submit button, when the form is post  among all the form keys **there isn't** the q-btn name. 

Having the name of the button at action side is useful to determine which button has been  pressed.

**Comment by rstoenescu** [comment, marked as answer]
An additional property is not needed since you can simply add the name attribute to your buttons.

**Reply by JTrotta** [reply]
I did it.
But using q-form tag and submitting it,  the button is not listed into key-form list on server side.
While using form tag and submitting it, the button is listed into key-form list on server side

q-form is much more useful than form tag, because all form components could be validate on client side.
Can you help me?

**Comment by pdanpdan** [comment]
please make a codepen with both situations
